lm_eval/models/ibm_watsonx_ai.py

```diff
@@ -252,27 +227,24 @@ def generate_until(self, requests: List[Instance]) -> List[str]:
         Returns:
             List[str]: A List of generated responses.
         """
-        requests = [request.args[0] for request in requests]
+        requests = [request.args for request in requests]
         results = []
-        batch_size = 5
 
-        for i in tqdm(
-            range(0, len(requests), batch_size),
-            desc=f"Running generate_until function with batch size {batch_size}",
+        for request in tqdm(
```

Comment:
Should document that we don't use batching due to (current) indeterminism of batching.

Reply:
I've left a note in the code for that.